hwy/ops/arm_sve-inl.h

```diff
@@ -1509,6 +1509,11 @@ HWY_API svint32_t DemoteTo(Simd<int32_t, N, kPow2> d, const svfloat64_t v) {
       NAME(HWY_SVE_D(BASE, BITS, N, kPow2) /* d */, HWY_SVE_V(int, BITS) v) { \
     return sv##OP##_##CHAR##BITS##_s##BITS##_x(HWY_SVE_PTRUE(BITS), v);       \
   }                                                                           \
+  template <size_t N, int kPow2>                                              \
```

Comment:
Would be nice to add a comment `/* unsigned */` and perhaps also above `/* signed */`, it wasn't immediately visible what this change was doing.

Reply:
Done. And thank you for pointing my attention to this file - there was a bug.